.github/dependabot.yml

```
@@ -19,6 +19,11 @@ updates:
           patterns:
             - "pulp-cli"
             - "pulp-glue"
+       pulp-deb-deps:
```

This is just naming, but I think it is technically more accurate/more consistent with other naming elsewhere:
```suggestion
       pulp-cli-deb-deps:
```

---

I'll take that